content/tutorial/tutorial.md

```diff
@@ -12,130 +12,130 @@ redirect_from:
   - "docs/tutorial-zh-CN.html"
 ---
 
-This tutorial doesn't assume any existing React knowledge.
+Aquest tutorial no assumeix cap coneixement previ sobre React.
 
-## Before We Start the Tutorial {#before-we-start-the-tutorial}
+## Abans de començar el tutorial {#before-we-start-the-tutorial}
 
-We will build a small game during this tutorial. **You might be tempted to skip it because you're not building games -- but give it a chance.** The techniques you'll learn in the tutorial are fundamental to building any React apps, and mastering it will give you a deep understanding of React.
+Anem a construir un petit joc durant aquest tutorial. **Hauràs estar temptat a obviar-perquè tu no estàs construint jocs en el dia a dia, però dóna-li una oportunitat.** Les tècniques que aprendràs en el tutorial són fonamentals per a construir qualsevol aplicació de React, i dominar-et donarà una entesa profunda de React.
```

Comment:
This sentence "Hauràs estar temptat a obviar-perquè tu no estàs construint jocs en el dia a dia, però dóna-li una oportunitat" doesn't convey the exact meaning as in English. I would suggest using: "Podries estar temptat a no seguir el tutorial perquè fer jocs no és lo teu -- però dóna-li una oportunitat."